src/main/java/com/toptrumps/core/card/Card.java

```diff
@@ -24,6 +24,8 @@ public Card() {
     public Card(String name, ArrayList<Attribute> attributes) {
         this.name = name;
         this.attributes = attributes;
+        highestAttributes = new ArrayList<>();
+        findHighestAttributes();
```

Comment:
You have removed the lazy loading logic. Why not make this computation only when we need it? Remember that only AI Players need to auto-select the highest attributes.